docs/gettinginvolved/plugins/building-driver.rst

```diff
@@ -0,0 +1,161 @@
+.. _build-flocker-driver:
+
+================================
+Building and Testing your Driver
+================================
+
+This document will show you how to implement a Flocker storage driver.
+Your driver will be a Python 2.7 library providing a class implementing the `flocker.node.agents.blockdevice.IBlockDeviceAPI <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/blockdevice.py>`_ interface.
+
+The best way to build your driver is to model it on the canonical implementations provided by the ClusterHQ team.
+These drivers include:
+
+* `OpenStack Cinder <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/cinder.py>`_
+* `Amazon EBS <https://github.com/ClusterHQ/flocker/blob/master/flocker/node/agents/ebs.py>`_
+
+After you have implemented the driver, you will need to test your implementation, and ClusterHQ provide a number of test suites to help you do this.
+Passing all tests ultimately results in certification of your driver to work with Flocker.
```

Comment:
What does this mean? Will I receive a certificate, or will I just be pretty sure it works with Flocker?

If it is the latter, maybe something like: "Passing all tests provides guarantees that your driver works with Flocker."